qiime2/sdk/usage.py

```diff
@@ -317,6 +728,15 @@ def _get_records(self):
 
 
 class DiagnosticUsage(Usage):
+    """
+    Reference implementation of a Usage driver.
+
+    Used to generate information for testing the Usage API.
+
+    See Also
+    --------
+    qiime2.plugin.tests.TestUsage : Unit tests using this driver.
```

Comment:
Looks like `TestUsage` isn't a suite in this module (although that probably should be its name, can you rename in this PR?).